tests/task.py

```diff
@@ -0,0 +1,123 @@
+from os import getenv as os_getenv
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torch.utils.data import DataLoader, Subset
+
+from .model import LinearMLP
+from .dataset import XORDataset, ExtraXORDataset
+
+
+def use_cuda():
+    if int(os_getenv('CPU_ONLY', '0')):
```

Comment:
This is a good idea. A possible improvement would be to make it a pytest command-line argument to avoid using environment variables